tests/install/steps/test_expansion.py

```diff
@@ -37,16 +37,12 @@ def declare_node(
 
 @when(parsers.parse('we deploy the node "{node_name}"'))
 def deploy_node(host, ssh_config, version, node_name):
-    accept_ssh_key = [
```

Comment:
Seems unrelated to this PR.

And if this is really useless, then the doc needs an update as well: https://metal-k8s.readthedocs.io/en/development-2.6/installation/expansion.html#id3

So maybe this should be done in a dedicated PR (and keep this one only about py3 migration)

Reply:
Yes you right, for the doc I think it's still good to have it as a pre-check for expansion

Reply:
I updated the documentation but I think I will keep it in this PR since it's changes needed for Python3 migration even if it's not directly related

Reply:
Maybe you could replace this command with the one suggested in docs (using `--raw-shell`)? I think we added it here to make sure the test would fail early if there was any issue with the key, and also because it's good practice to do in tests exactly what we have in docs.

Reply:
For me this one was here just because it was needed at the beginning to accept the ssh public key of the new node.
Anyway right I can add this check here like in the doc